packages/components/src/InputText/InputText.tsx

```diff
@@ -30,6 +30,8 @@ interface BaseProps
       | "defaultValue"
       | "prefix"
       | "suffix"
+      | "toolbar"
+      | "toolbarVisibility"
```

Comment:
do these make sense here? like yeah it makes it easy to pass them down to the underlying FormField because it knows about them, but these are input text specific

slightly worried that if another component directly uses `FormFieldProps` without `Pick`ing, then it'll inherit these things that won't make sense

Reply:
the pattern for at least the existing Input{Thing} is to `Pick` so I think you're good?

assuming 
- if someone makes a new component that pulls FormField in, they'd go "no toolbar for me please" when picking in that usage
- the `Pick` pattern is something we can either reliably catch in reviews, or have tooling around?

Reply:
One thought we could have is a `FormFieldInternalProps` that we don't export when publishing the package. We could then expose all the other FormField props we currently do. I don't think this is something we should do because `FormField` is very much an internal component that you should know what you are doing if you are going to use it outside of Atlantis. For example it is marked private
<img width="264" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/0360de96-9a73-4222-8423-4ea7b9476e19">